### PR TITLE
Issue 1837 : Make common readme more accurate

### LINKED ISSUE #1837: Fix readme of common
Fix readme of common to indicate that common is for classes that are not Pravega  specific.

### PR DESCRIPTION
**Pull request by moizsj**
**Change log description**
Make the common readme more accurate

**Purpose of the change**
Fixes #1837

**What the code does**
Fixes #1837

**How to verify it**
Read the readme.

### CODE REVIEW

**Comment by moizsj** [comment]
@fpj pls review and leave a pointless comment and I will make changes per the comments.

File: common/Readme [hunk 1]
```diff
@@ -1 +1 @@
-Code shared between client and server
+Code shared between client and server.
```
Comment (tkaitchuck) [inline common/Readme]: While your at it, this line is actually inaccurate. It should read:
General purpose utility classes. Not Pravega specific.

**Comment by moizsj** [comment]
@tkaitchuck thanks. fixed.

File: common/Readme [hunk 2]
```diff
@@ -1 +1 @@
-Code shared between client and server
+General purpose utility classes. Not Pravega specific.
```
Comment (fpj) [inline common/Readme]: I think it should be a common between `classes` and `not` rather than period.

**Comment by fpj** [comment]
No matter how simple the fix is, it should have an associated issue. The title of the pull request should start with `Issue ###: brief summary`, where `###` is the issue number.

Also, please do not delete the sections that are populated by default in the description box. It is fine if you end populating it with trivial statements, but always populate them. A couple of points about that:

- The content of `Changeling description` is in principle what we add to the text box of the merge message (we squash and merge), so it should be a succinct list of key changes that are being merged. It is at the discretion of the committer merging to make changes to that text, like correcting typos clarifying, etc.
- In `What the code does`, we typically add `Fixes #<issue number>` because that causes the issue to be closed along with the pull request.

Finally, we use the assignee to determine who is supposed to have a look at the pull request next. For example, when you create the pull request, the assignees are the reviewers. Once it has been reviewed, the assignees field should be turned to the developed. We haven't been following this procedure for assignees very strictly, but we should as it helps you to determine what are the pull requests you need to have a look at next.

**Comment by moizsj** [comment]
@fpj addressed all your comments except the assignee part. it seems I cant do that until I am made a collaborator on the project.

**Pull request merged**